### PR TITLE
bump for doctrine orm 2 -> 3, dbal 3 -> 4 through roave psr container

### LINKED ISSUE #264: Implement doctrine 3

### PR DESCRIPTION
**Pull request by pinclau**

### CODE REVIEW

**Review by alexmerlin — COMMENTED** [review]
Any idea why the `uuid` column has moved towards the end of the table, before the `created` column?
I guess Doctrine3 treats differently an entity when it extends another entity.

**Comment by pinclau** [comment]
> Any idea why the `uuid` column has moved towards the end of the table, before the `created` column? I guess Doctrine3 treats differently an entity when it extends another entity.

Yes, because those columns comes from AbstractEntity, we need to declare this class with MappedSuperclass attribute, and if we want the `uuid` to appear first and timestamps last, we need to split them.

**Comment by pinclau** [comment]
@alexmerlin The problem from github actions is dot-data-fixtures who request doctrine/data-fixtures ^1.5. support for orm 3 and dbal 4 was added from 1.7.
We need to disquss how we aproach this to be sure that it works for api v4 and v5.

File: src/App/src/Entity/TimestampsTrait.php
Comment (alexmerlin) [inline src/App/src/Entity/TimestampsTrait.php:1]: the good: `updated` should get updated when the entity gets updated.
the bad: `updated` should stay `null` at entity creation.
@arhimede correct me if I'm wrong.

Comment (arhimede) [reply src/App/src/Entity/TimestampsTrait.php:1]: is not a tragedy if at the creation time, created and updated fields have the samr value
if this simplify the code, we should keep it that way

Comment (arhimede) [reply src/App/src/Entity/TimestampsTrait.php:1]: Second tought (or so)

We should not use ilogical stuff in database.
So at creation time, should be null the updated field

**Review by arhimede — APPROVED** [review]

**Pull request merged**